relayer/listener.go

```diff
@@ -20,10 +21,10 @@ import (
 )
 
 const (
-	maxSubscribeAttempts = 10
+	retrySubscribeTimeout = 10 * time.Second
 	// TODO attempt to resubscribe in perpetuity once we are able to process missed blocks and
 	// refresh the chain config on reconnect.
-	maxResubscribeAttempts = 10
+	retryResubscribeTimeout = 10 * time.Second
```

Comment:
We can remove `retry` from these names to be consistent with the timeout constants in other files.

Reply:
I thought about the way to differentiate retry timeout and operation timeout. Please see my comment https://github.com/ava-labs/icm-services/pull/593#discussion_r1899165757